programs/sample_enroll/sample_enroll.c

```diff
@@ -75,6 +75,12 @@ struct enroll_cert {
 #include <ziti/zitilib.h>
 
 int main(int argc, char **argv) {
+
+    if (argc != 2) {
```

Comment:
this should be `argc >= 3`

usage is `sample_enroll <JWT file> <ID file> [key_file] [cert_file]`

Reply:
You probably have meant `if (argc  < 3) -> prompt`?
JWT/ID files are mandatory.
key/cert files are optional.

Reply:
Fixed.